x/ecocredit/marketplace/msg_update_sell_orders.go

```diff
@@ -24,26 +26,55 @@ func (m MsgUpdateSellOrders) GetSignBytes() []byte {
 
 // ValidateBasic does a sanity check on the provided data.
 func (m *MsgUpdateSellOrders) ValidateBasic() error {
+	if len(m.Owner) == 0 {
+		return sdkerrors.ErrInvalidRequest.Wrap("owner cannot be empty")
+	}
+
 	if _, err := sdk.AccAddressFromBech32(m.Owner); err != nil {
-		return sdkerrors.ErrInvalidAddress
+		return sdkerrors.ErrInvalidRequest.Wrapf("owner is not a valid address: %s", err)
```

Comment:
i think we opted to keep ErrInvalidAddress as we did in #1140

Reply:
https://github.com/regen-network/regen-ledger/pull/1134/commits/6b5d995e5a77025f4155f0e081056b0152cff09f